src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/StringAttributeCollection.cs

```diff
@@ -44,7 +44,7 @@ internal void FromString(string list)
                 foreach (string item in items)
                 {
                     string trimmedItem = item.Trim();
-                    if (trimmedItem.Length != 0) Add(item.Trim());
+                    if (trimmedItem.Length != 0) Add(trimmedItem);
```

Comment:
I wonder if this even can be improved further using the `Trim` extension of the ReadOnlySpan<char>?

@stephentoub

Reply:
How?  It's storing the result into a collection.  You can't store a span into such a collection.